tests/testthat/test-public_api.R

```diff
@@ -155,3 +155,28 @@ test_that("insufficient R version returns error", {
   expect_error(stop_insufficient_r_version())
 })
 
+context("public API - Rnw in style_file()")
+
+test_that("styler can style Rnw file", {
+  capture_output(expect_false({
```

Comment:
So for testing, I suggest we stick to the infrastructure we established before for consistency, i.e. use `test_collection()`. I think this would require to adapt `test_collection` to reckon the `.Rnw` pattern: 
https://github.com/r-lib/styler/blob/master/R/testing.R#L30

Reply:
This is used almost everywhere, e.g. here: https://github.com/r-lib/styler/blob/master/tests/testthat/test-indention_curly.R.

Reply:
Sorry, did not see the tests below. -.-

Reply:
😄